packages/testharness/src/index.js

```diff
@@ -10,6 +10,7 @@ import classNames from 'classnames';
 import createDeferred from 'p-defer-es5';
 import expect from 'expect';
 import lolex from 'lolex';
+import Observable from 'core-js/features/observable';
```

Comment:
do we need this in the es5 bundle too, by any chance? or this is just for testing?

Reply:
It is just for the test harness. The new test need to use Observable.